external/webserver/http_client.c

```diff
@@ -305,6 +339,9 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 						if (strcmp(key, "Sec-WebSocket-Key") == 0) {
 							strncpy((char *)client->ws_key, value, WEBSOCKET_CLIENT_KEY_LEN);
 						}
+						if (strcasecmp(key, "Keep-alive") == 0) {
```

Comment:
Most of other keys are compared with considering the case sensitive.
But only for this, you compared regardless of case sensitive.
Is there any special reason?

Reply:
client can send keep-alive in different case

Reply:
Okay. Then other keys are fixed?

Reply:
No checking other case in this PR